src/components/TodoFilter.tsx

```diff
@@ -0,0 +1,68 @@
+import classNames from 'classnames';
+import { StatusFilter } from '../types/StatusFilter';
+
+type Props = {
+  status: StatusFilter,
+  handleStatusChange: (status: StatusFilter) => void,
+  countActiveTodos: number,
+  countCompletedTodos: number,
+  handleClearCompleted: () => void,
+};
+
+export const TodoFilter: React.FC<Props> = ({
+  status,
+  handleStatusChange,
+  countActiveTodos,
+  countCompletedTodos,
+  handleClearCompleted,
+}) => {
+  return (
+    <footer className="todoapp__footer" data-cy="Footer">
+      <span className="todo-count" data-cy="TodosCounter">
+        {`${countActiveTodos} items left`}
```

Comment:
If there is one item, there should not be a plural in the text.

Reply:
> If there is one item, there should not be a plural in the text.

I can't change it because the tests failed